index.html

```diff
@@ -78,6 +79,45 @@ <h1>
         </section>
 
 
+<section class="modal-pop active id="pop-up">
+        <div class="card-item-modal">
+            <div class="main-text">
+                <div class="modal">
+                    <h1 class="title">Tonic</h1>
+                    <button data-close-btn class="close-x ">&times;</button>
+                </div>
+                <ul class="frame-2">
+                    <li class="top-title">Canopy</li>
+                    <li class="details"><img src="./images/Counter.png" alt="Image missing"></li>
+                    <li class="details">Back End Dev</li>
+                    <li class="details"><img src="./images/Counter.png" alt="Image missing"></li>
+                    <li class="details">2015</li>
+                </ul>
+                <div class="images-tags">
+                    <a href="#">
+                        <img class="img1" alt="Image missing">
+                    </a>
+                </div>
+
+                <p class="description">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="tags">
```

Comment:
- [ ] Also, the requirements say you should render the cards using a dynamic HTML using the details that will be in an array as object properties. You're to parse the contents from the array object, then use javascript to render it. this is made so, so that it gets easy to add or remove a project in the future:

# Sample way to do it:

- Array object

```const dataCards = [
  {
    id: 0,
    url: './assest/ConfrencePage.png',
    title: 'Conference-Page-Design',
    description:
      'Final Project from HTML and CSS course. An imaginary confrenec page app that details a cloud computing event',
    tecnologies: {
      tech1: 'JavaScript',
      tech2: 'css',
      tech3: 'Figma',
      tech4: 'html',
    },
    linkVersion: 'https://shedrack-sunday.github.io/Conference-Page-Design/',
    linkSource: '/Shedrack-Sunday',
  },
  {
    id: 1,
    url: './assest/leaderboard.png',
    title:
      ' A board built, utilizing callbacks and promises, proper ES6 syntax. ES6 modules to write modular JavaScript. Use webpack to bundle JavaScript.',
    description: `The leaderboard is a page that let users ADD books based on authors and the respective books.
    it fetches the data from an external API, then displayes the rsulting information on screen.`,
    tecnologies: {
      tech1: 'Javastcrip',
      tech2: 'webpack',
      tech3: 'CSS',
      tech4: 'GIthiub Actions',
    },
    linkVersion: 'https://github.io/Leaderboard/dist/',
    linkSource: 'https://github.com',
  },
  {
    id: 2,
    url: './assest/books.png',
    title: 'Awesome Books',
    description: ' A Book app to add authors and their books. Built with JavaScript, Html, and CSS.  ',
    tecnologies: {
      tech1: 'Webpack',
      tech2: 'Html',
      tech3: 'JavaScript',
      tech4: 'css',
    },
    linkVersion: 'https://github.io/',
    linkSource: 'https://github.com/',
  },
  ```
  
  # Sample rendering with Javascript:
  ```
  const cardWorkDetailsMobile = (index) => `
  <article class="card-work-details-mobile">
          <button type="button" class="btn-close-details">
              <img src="./assest/Icon-Cancel.svg" alt="cancel icon">
          </button>
          <div class="card-image-details-mobile">
              <img src=${dataCards[index].url} alt="a guy with a backpack in mountain">
          </div>
          <h2 class="card-title-mobile">
              ${dataCards[index].title}
          </h2>
          <ul class="card-tags-mobile">
              <li class="tag-mobile">${dataCards[index].tecnologies.tech1}</li>
              <li class="tag-mobile">${dataCards[index].tecnologies.tech2}</li>
              <li class="tag-mobile">${dataCards[index].tecnologies.tech3}</li>
              <li class="tag-mobile">${dataCards[index].tecnologies.tech4}</li>
          </ul>
          <p class="card-paragraph-mobile">
            ${dataCards[index].description}
          </p>
          <div class="card-button-mobile">
                  <a href=${dataCards[index].linkVersion} class="btn-card-details-mobile">
                  See Live
                  <img src="./assest/" alt="see live icon">
                  </a>
                  <a href=${dataCards[index].linkSource} class="">
                  See Sources
                  <img src="">
                  </a>
          </div>
          <script src="index.js"></script>
  </article>
  ```